script.js

```diff
@@ -0,0 +1,75 @@
+var evenOdd;
+var myFuncCalls = 0;
+if (myFuncCalls%2===0){
```

Comment:
Also as we talked about yesterday, I suggest you use a linter to auto-check your code for style issues (e.g. using spaces between `%` and `===` etc.)

There are packages for atom (search google for atom linter and JSlint / jshint), or you can use jshint.com or jslint.com.